### ISSUE TITLE
Add more examples to documentation

### ISSUE THREAD

**Issue by tcbrindle** [tahonermann/text_view#32]
While the reference documentation is comprehensive, it would be very handy to have some high-level examples of what the library can do and why I would want to use it.

For example, Howard Hinnant's excellent [date library](https://github.com/HowardHinnant/date) has a page of "examples and recipes" available in its [wiki](https://github.com/HowardHinnant/date/wiki/Examples-and-Recipes).

**Comment by tahonermann** [comment]
I agree this would be nice to have.  The only examples I've posted so far are those in the overview section of the readme (https://github.com/tahonermann/text_view#overview).  Do you have any suggestions for good examples?  The current feature set is pretty limited (I plan to start working on new features in the next couple of weeks), and I have struggled to come up with interesting examples because of that.  Any chance you would be interested in drafting some wiki pages?

**Comment by tahonermann** [comment]
Hi Tristan.  I finally got around to throwing together a few wiki pages loosely following Howard's work as you suggested.  Does this match what you had in mind?  Any suggestions for improving it would be welcome.

https://github.com/tahonermann/text_view/wiki
